app/build.gradle

```diff
@@ -23,11 +23,16 @@ android {
     lintOptions {
         abortOnError false
     }
+
     compileOptions {
         targetCompatibility 1.8
         sourceCompatibility 1.8
     }
 
+    dexOptions {
+        jumboMode = true
+    }
+
```

Comment:
Do we need this? What problem does it solve?

Reply:
It's been a bit since I did this, but I believe this was necessary since we pull in more dependencies now. I think I was experiencing some issues during the build?

Reply:
Hmm.. at least here I don't run into any issues when not using jumboMode.